codice/ddf
# DDF-4461 Migrate Map Components to React - map-info.view.js

#### What does this PR do?
Migrates the map info component to React
#### Who is reviewing it? 

@BernardIgiri 
@adimka 
@Bdthomson 
@andrewzimmer 
#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@djblue 

#### How should this be tested?
Ensure previous functionality was preserved. This includes:
 - When the user hovers over a point on the map, coordinates formatted based on user prefs will be displayed on the screen. When off map, nothing is displayed.
 - On search, hovering over a search result on the map will display metacard information in addition to coordinates(the attributes displayed are set through the admin ui - https://github.com/codice/ddf/pull/3985)

#### Any background context you want to provide?
#### What are the relevant tickets?
For Jira:
[DDF-4461](https://codice.atlassian.net/browse/DDF-4461)

For GH Issues:
Fixes: #____

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
